Renderer.js

```diff
@@ -0,0 +1,47 @@
+
+
+class Renderer {
+    constructor(data)
+    {
+        this.data = data
+    }
+
+    renderData = function () {
+        const ourData = this.data
```

Comment:
maybe instead make 5 methods inside the Renderer class. Then it will be a lot cleaner